backend/src/main/java/peer/backend/controller/SignUpController.java

```diff
@@ -60,7 +58,7 @@ public ResponseEntity<Object> signUp(@Valid @RequestBody UserInfo info) {
     public ResponseEntity<Object> withdrawal(@RequestBody String password,
         Authentication authentication) {
         User user = User.authenticationToUser(authentication);
-        if (this.memberService.verificationPassword(password, user.getPassword())) {
+        if (!this.memberService.verificationPassword(password, user.getPassword())) {
```

Comment:
부정 조건문은 의미 파악이 직관적이지 않아 해당 메서드가 뭘 반환 하는지 확인이 어려운 경우가 있어요.
사용하는 곳이 이 곳 뿐이라 시간 되실 때 같이 바꾸면 좋을 것 같습니다.

```suggestion
        if (this.memberService.verificationPassword(password, user.getPassword())) {
            this.memberService.deleteUser(user);
            return ResponseEntity.ok().build();
        }       
        throw new UnauthorizedException("비밀번호가 잘못되었습니다!");
```

```Java
// MemberService.java :: 55
    public boolean verificationPassword(String input, String password) { // 비밀번호 검증 메서드
        if (encoder.matches(input, password)) { // 입력값과 비밀번호가 일치하다면
            return true; // true 를 반환
        } // 아니라면
        return false; // false 반환
    }
```

Reply:
개인적으로 return이 마지막에 오는 함수 구조를 선호해서요.
그것보다 return encoder.matches(input, password)를 하면 됐는데 제가 저기를 왜 저렇게 했는지 모르겠네요 ㅎㅎ
피드백 감사합니다!